### DISCUSSION TITLE
Any estimates for v1.0.0 release?

### DISCUSSION THREAD

**Discussion by iNikem** [open-telemetry/opentelemetry-js-api#26] [Q&A] [answered]
The titles says it all :)

**Comment by dyladan** [comment, marked as answer]
Probably not for a few weeks at least. The SDK is still not at RC release yet and I would expect at least a couple weeks of the API and SDK RCs running together before releasing a 1.0.0 of the API or both. We hope to have the SDK RCd this week if at all possible but at this point it seems unlikely which means next week.

**Reply by iNikem** [reply]
Thanks!
